test/cluster/config_test.go

```diff
@@ -0,0 +1,419 @@
+// Copyright IBM Corp. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cluster
+
+import (
+	"crypto/tls"
+	"encoding/pem"
+	"github.com/hyperledger-labs/orion-server/pkg/crypto"
+	"github.com/hyperledger-labs/orion-server/pkg/server/testutils"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hyperledger-labs/orion-server/pkg/types"
+	"github.com/hyperledger-labs/orion-server/test/setup"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	baseNodePort = 6100
+	basePeerPort = 7100
+}
```

Comment:
these vars are already in an init() in the basic_test.go file, no need to do this here and not in catchup_test.go either. init() is per package.